pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1550,5 +1554,55 @@ func TestAlgorithmProviderCompatibility(t *testing.T) {
 			}
 		})
 	}
+}
 
+func TestPluginConfigV1Alpha1(t *testing.T) {
+	// This test ensure the correctness of convert v1alpha1.PostFilter config to PreScore
+	testcases := []struct {
+		name        string
+		plugins     *config.Plugins
+		wantPlugins *config.Plugins
+	}{
+		{
+			name: "Test v1alpha1.PostFilter to config.PreScore",
+			plugins: func() *config.Plugins {
+				cfgv1alpha1 := v1alpha1.KubeSchedulerConfiguration{}
+				scheme.Scheme.Default(&cfgv1alpha1)
+				cfgv1alpha1.Plugins = &v1alpha1.Plugins{}
+				cfgv1alpha1.Plugins.PostFilter = &v1alpha1.PluginSet{
+					Enabled: []v1alpha1.Plugin{
+						{Name: "TestPostFilter1"},
+						{Name: "TestPostFilter2"},
+						{Name: "TestPostFilter3"},
+					},
+				}
+
+				cfg := config.KubeSchedulerConfiguration{}
+				if err := scheme.Scheme.Convert(&cfgv1alpha1, &cfg, nil); err != nil {
```

Comment:
This is the wrong location for this test.
In this file, you should be exercising the creation of `scheduler.New` from a plugin configuration. However, we already use parsed versions of the component config when calling `scheduler.New`. So maybe we don't need this test.

Rather, we could have a test in `cmd/kube-scheduler/app/options/options_test.go`

@ahg-g ?

Reply:
Yes, I think it's better to put this into options_test too. I did found some example that  helps me with writing this ut at options_test.go

Reply:
moving it to options_test.go sounds good, removing it completely is also fine since we have coverage in conversion_test.go